src/sdk/cart/useBuyButton.ts

```diff
@@ -1,12 +1,34 @@
 import { useCallback } from 'react'
+import type { AddToCartEvent, Item as AnalyticsItem } from '@vtex/store-sdk'
+import { sendAnalyticsEvent, useSession } from '@vtex/store-sdk'
 
 import { useUI } from '../ui'
 import { useCart } from './useCart'
 import type { CartItem } from './useCart'
 
-export const useBuyButton = (item: Omit<CartItem, 'id'> | null) => {
+type AdditionalItemProperties = {
+  product_reference_id: string | null
+  sku_reference_id: string | null
+  sku_name: string | null
+}
+
+interface VTEXAddToCartEvent extends AddToCartEvent {
+  data: AddToCartEvent['data'] & {
```

Comment:
I figured out that on store-sdk we export an interface called `AddToCartData` that can replace this call. I think it'd be cleaner :)

```suggestion
  data: AddToCartData & {
```

Reply:
Great. I believe this solves it? https://github.com/vtex-sites/base.store/pull/53/commits/e84887be8e1813e5866d198221006e7a930f3990